tests/migration_test.go

```diff
@@ -4341,7 +4341,7 @@ var _ = Describe("[rfe_id:393][crit:high][vendor:cnv-qe@redhat.com][level:system
 				}, 90*time.Second, 3*time.Second).Should(BeTrue(), fmt.Sprintf("tsc frequency topology hints are expected to exist for vmi %s", vmi.Name))
 			}
 
-			It("invtsc feature exists", func() {
+			It("invtsc feature exists", decorators.Invtsc, func() {
```

Comment:
I think that `[test_id:6842]` is also relevant for tsc

Reply:
Done.